lib/utils.js

```diff
@@ -36,20 +56,30 @@ function getChangelogSectionIndex (lines) {
 }
 
 /**
+ * Get the given key from process.env, substituting "undefined" for the real undefined
+ * @param {String} key - the environment variable we want to get
+ * @returns {*} whatever is at process.env[key] with the one exception of "undefined" being translated to undefined
+ */
+function getEnv (key) {
+  const value = process.env[key]
+  return (value === 'undefined') ? undefined : value
```

Comment:
I see now you check against the string `undefined` whereas before we used `||`, checking against the value actually being undefined/null/0. Does this mean defaults never actually worked before?